tox.ini

```diff
@@ -59,17 +62,6 @@ deps = {[with-debug-tools]deps}
 [testenv:py34_debug]
 deps = {[with-debug-tools]deps}
 
-[testenv:py27_profile]
```

Comment:
Since pytest-profiling is buggy, and since this testenv is rarely used, I just removed it.